index.html

```diff
@@ -1655,6 +1655,33 @@ <h2>Contexts and Vocabularies</h2>
 Verifiable Credentials v2.0: Vocabularies</a>.
         </p>
 
+        <section>
+          <h3>Context Injection</h3>
+
+          <p>
+The `@context` property is used to ensure that implementations are using the
+same semantics when terms in this specification are processed. For example, this
+can be important when properties like `type` are processed and its value, such
+as `DataIntegrityProof`, are used.
+          </p>
+
+          <p>
+If a `@context` property is not provided in a document that is being secured or
```

Comment:
```suggestion
If an `@context` property is not provided in a document that is being secured or
```

Reply:
I don't think the `@` symbol is pronounced, so I don't think this needs an `an`

Reply:
If the `@` isn't pronounced, how do you tell the difference between `context` and `@context`?

Reply:
Also, if we're going to use `a @context` here, line 1672 must be updated to match, as well as numerous instances across other documents.

I do noting that even [the JSON-LD 1.1 spec](https://www.w3.org/TR/json-ld11/#scoped-contexts) is inconsistent about this! At the link, you'll find `include a @context property` in the first paragraph of text in §4.1.8, and `Defining an @context` in the titles of `Example 45` and `Example 46`.

That said, searching for `an @context` throughout that spec finds 6 instances, while `a @context` finds only 1 instance. I think that's a clear enough indication of the intention of the spec authors, and I'll continue to press for `an @context` here.

Reply:
The omission was an error. I pronounce the "@" symbol and always strive to put "an @" when writing. I agree with the world that @TallTed wants to live in and denounce the grammatically lawless world, devoid of strange English rules that take a half a lifetime to learn, that @dmitrizagidulin is attempting to lull us into. :P